### DISCUSSION TITLE
Webpage element on slide

### DISCUSSION THREAD

**Discussion by ajulvestad** [reaby/infoscreen3#22] [Ideas]
Add webpage embed component in slide editor.

We usually have slides made up of different items. E.g. a slide consisting of a calendar entry "next events" and other things. To be able to embed webpages (preferrably with crop and zoom) would extend the features a lot.

It would also then be possible to have the same consistent feel - a webpage with transparent background, so the clock and background animation is still shown.

**Comment by reaby** [comment]
I'm not sure if I can do custom embeds easily... But you can always like you said: have a webpage with transparent background to achieve this.
set at the css:
```css
html, body {
    background: none !important; 
}
```
